@types/modules/pm/groups.d.ts

```diff
@@ -0,0 +1,20 @@
+declare interface PMGroupUpdateRequest {
+    prizeCategoryId: string;
+    prizeCatagoryName?: string;
+    prizeCategoryDescription?: string;
+    prizeCategoryOrder?: Array<string>;
+  }
+
+  declare interface PMGroupCreateRequest extends PMGroupUpdateRequest {
+    prizeCatagoryName: string;
+    prizeCategoryDescription: string;
+    prizeCategoryOrder: Array<string>;
+  }
+
+  declare interface PMGroup extends PMGroupCreateRequest {
+    prizeCategories: { [key: string]: string };
```

Comment:
If the above fields are optional, then they get declared here (similar to what we did with id).
```suggestion
    prizeGroupDescription: string;
    prizeCategoryOrder: string[];
```